tests/unit/test_utils.py

```diff
@@ -329,3 +329,16 @@ def test_check_before_after_census_day(self):
         self.after_census_day(2021, 3, 22)
         self.before_census_day(2021, 3, 21)
         self.before_census_day(2021, 3, 20)
+
+    def test_client_ip(self):
+        valid_request = {'client_ip': '192.168.0.0, 35.190.0.0, 35.191.10.0'}
+        spoofed_request = {'client_ip': '200.30.20.10, 192.168.0.0, 35.190.0.0, 35.191.10.0'}
+        invalid_request = {'client_ip': 'GHT.RGT.ED.WS, 192.168.0.0, 35.190.0.0, 35.191.10.0'}
+        built_valid = View.single_client_ip(valid_request)
+        built_invalid = View.single_client_ip(invalid_request)
+        built_spoofed = View.single_client_ip(spoofed_request)
+        expected_valid = '192.168.0.0'
+        expected_empty = ''
+        self.assertEqual(built_valid, expected_valid)
+        self.assertEqual(built_spoofed, expected_valid)
```

Comment:
Should a spoofed request come through as expected_valid? I'm not sure I understand this.